.github/contributing/commits.md

```diff
@@ -16,41 +15,31 @@ to be in a specific format, so they can be parsed.
 
 ### Type
 
-For the type part of the commit, we use the ones defined by the
-[Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
+The commit type part must be one defined by the [Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
 
 - **build**: Changes that affect the build system or external dependencies.
 - **ci**: Changes to our CI configuration files and scripts.
-- **chore**: Changes that you don’t want to appear in the change-log. For example this can be used
-  to fix the change-log if any release goes wrong.
+- **chore**: Changes that you don’t want to appear in the change log. For example, this can be used to fix the change log if any release goes wrong.
```

Comment:
Is it `change-log`, `change log`, or `changelog`? I usually see it as just one word.

Reply:
Agree

Reply:
Linguistically speaking it should be change log (as it's a log of changes), but [GitHub has gone for changelog](https://github.com/conventional-changelog/standard-version/issues/299). Example: https://docs.github.com/en/graphql/overview/changelog

I'll unify throughout.